docs/guidance/rayjob.md

```diff
@@ -0,0 +1,129 @@
+## Ray Job (alpha)
+
+> Note: This is the alpha version of Ray Job Support in KubeRay. There will be ongoing improvements for Ray Job in the future releases.
+
+### Prerequisite
+
+* Ray 1.10 and above.
+* KubeRay v0.3.0 or master
+
+### What is a RayJob?
+
+The RayService is a new custom resource (CR) supported by KubeRay in v0.3.0.
```

Comment:
Find and replace
RayService -> RayJob
in this doc.

Reply:
Yeah will do